pkg/api/v1beta1/types.go

```diff
@@ -726,10 +844,12 @@ type NodeResources struct {
 type ResourceName string
 
 const (
-	// CPU, in cores. (floating point w/ 3 decimal places)
+	// CPU, in cores. (500m = .5 cores)
 	ResourceCPU ResourceName = "cpu"
-	// Memory, in bytes.
+	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	ResourceMemory ResourceName = "memory"
+	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
+	ResourceStorage ResourceName = "storage"
```

Comment:
The problem with this approach is that, for storage, IOPS is in practise often more important than space.  Contrast with memory, where people typically only care about amount.  It boils down to how averse we are to changing the API down the road, as I'm very certain that we'll need to distinguish storage.space from storage.iops pretty soon.  Presumably it's much less disruptive to start with "storage.space", and add "storage.iops" etc down the road, than it is to start with "storage", and then deprecate it in favor of "storage.space" and "storage.iops" in the near future.

Happy to defer to your judgement here bgrant0607@, just wanted to make sure that my original comment was clear to all.